src/backend/gl/src/device.rs

```diff
@@ -1114,12 +1114,22 @@ impl d::Device<B> for Device {
             }
 
             if let Some(depth_stencil) = subpass.depth_stencil {
+                let aspects = match &attachments[depth_stencil] {
```

Comment:
A little pet peeve of mine (sorry!) - could we please not match on a reference?
There is no need to take advantage of the extra sugar here, we'd rather want explicit code, i.e. `match attachments[depth_stencil] {...}`.

Reply:
Done.